source/assets/stylesheets/landing.scss

```diff
@@ -11,8 +12,6 @@
   margin-bottom: 37px;
 
   @media screen and (min-height: 775px) and (min-width: $tablet-width) {
-    margin-top: 0;
-    margin-bottom: 0;
```

Comment:
These were causing weird spacing issues on large desktops, and it looks fine or better without them :man_shrugging: